pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -756,11 +756,21 @@ func (npc *NetworkPolicyController) appendRuleToPolicyChain(iptablesCmdHandler *
 	if dPort != "" {
 		args = append(args, "--dport", dPort)
 	}
-	args = append(args, "-j", "ACCEPT")
-	err := iptablesCmdHandler.AppendUnique("filter", policyChainName, args...)
+
+	markComment := "rule to mark traffic matching a network policy"
+	markArgs := append(args, "-j", "MARK", "-m", "comment", "--comment", markComment, "--set-xmark", "0x10000/0x10000")
```

Comment:
I take it that you chose this value, because it's above this range: https://github.com/cloudnativelabs/kube-router/blob/4d6b0b818fc6209142c7ca91b8774d4573de1349/pkg/controllers/proxy/network_services_controller.go#L1676 ? Is there anything else in kube-router than deals with marks?

Reply:
No other use of Markings. Just choose random numbers.